server/graphql/v1/mutations/orders.js

```diff
@@ -17,6 +17,18 @@ import {
   getNextChargeAndPeriodStartDates,
   getChargeRetryCount,
 } from '../../../lib/subscriptions';
+import LRU from 'lru-cache';
+
+// Create Orders Cache Object to limit FromCollective orders to up to 10 orders/hour
+const ordersFromCollectiveCache = LRU({
+  max: 1000,
+  maxAge: 1000 * 60 * 60, // we keep it max 1 hour
+});
+// Create Orders Cache Object to limit pair (FromCollective,Collective) orders to up to 2 orders/hour
+const ordersFromCollectiveAndCollectiveCache = LRU({
+  max: 1000,
+  maxAge: 1000 * 60 * 60, // we keep it max 1 hour
+});
```

Comment:
Why do we need to instanciate 2 cache instances?

Reply:
you're right! we don't. fixing it.